x-pack/plugins/uptime/public/components/certificates/cert_status.tsx

```diff
@@ -41,9 +62,18 @@ export const CertStatus: React.FC<Props> = ({ cert }) => {
     );
   }
 
+  const okRelativeDate = moment(cert?.not_after).fromNow(true);
+
   return (
     <EuiHealth color="success">
-      <span>{labels.OK}</span>
+      <span>
```

Comment:
This needs to be translated, something like:

```suggestion
      <span>
        {labels.OK}
        <DateText color="subdued" size="xs">
          <FormattedMessage
            id="xpack.uptime.certs.status.ok"
            defaultMessage=" for {okRelativeDate}"
            description={
              'Denotes an amount of time for which a cert is valid. Example: "OK for 2 days"'
            }
            values={{
              okRelativeDate,
            }}
          />
        </DateText>
      </span>
```

Alternatively you could extract the string to a label, but seems strange to do it for only one word.

Reply:
done.